server/topic.go

```diff
@@ -2392,13 +2424,20 @@ func (t *Topic) replyGetDel(sess *Session, asUid types.Uid, req *MsgGetOpts, msg
 	id := msg.Id
 	incomingReqTs := msg.Timestamp
 
+	asChan, err := t.verifyChannelAccess(toriginal)
+	if err != nil {
+		// User should not be able to address non-channel topic as channel.
+		sess.queueOut(ErrNotFoundReply(msg, now))
+		return types.ErrNotFound
```

Comment:
Return err?

Reply:
The user should get `types.ErrNotFound`, it's the correct response. `err` is also `types.ErrNotFound`, but if it changes the user may get less clear error message. I would rather keep it as is.